package.json

```diff
@@ -308,7 +308,6 @@
     "aws-sdk-client-mock": "3.0.0",
     "callsite": "1.0.0",
     "common-tags": "1.8.2",
-    "conventional-changelog-conventionalcommits": "7.0.2",
```

Comment:
are you sure? I think it's implicit used by semantic release via config.

Reply:
https://github.com/renovatebot/renovate/blob/d1898ed3fdf8ee5c807d88f4595a7f84fb320132/.releaserc#L38

I think that will cause this dep to be loaded .